R/bootstrap.R

```diff
@@ -500,7 +500,18 @@ tabPanel <- function(title, ..., value = title, icon = NULL) {
 #' @export
 #' @describeIn tabPanel Create a tab panel that drops the title argument.
 #'   This function should be used within `tabsetPanel(type = "hidden")`. See [tabsetPanel()] for example usage.
-tabPanelBody <- function(..., value = NULL, icon = NULL) {
+#'   `value` must be set when using `tabPanelBody`.
+tabPanelBody <- function(value, ..., icon = NULL) {
+  if (
+    missing(value) ||
+    nullOrEmpty(value) ||
```

Comment:
I think this is redundant as you're already checking that it's a character vector of length 1.